### PR TITLE
Few negative cases covered for previous features

### PR DESCRIPTION
**Pull request by naina-verma**
Test covered 
- Verify that workitems cannot be assigned to non-existent users
- Verify that the title field must not be unique
- Verify that the description field can have its contents deleted
- Check before while adding description the field should not display “undefine/null’
- Verify the HTML tags should be treated as text while writing description/title.
- Test description/title can be in other languages also acceptable
- Verify that all fields are read-only for these users 
- Verify that data is persisted and is not truncated if text fields receive data with a length greater that 255 characters.
- Verify visual display of data with a length greater that 255 characters.
- Verify if the date/time is in future it should say “in X time “

### CODE REVIEW

File: src/tests/work-item/work-item-list/page-objects/work-item-list.page.js [hunk 1]
```diff
@@ -272,6 +272,9 @@ class WorkItemListPage {
   get filterDropdown () {
     return  element(by.id("wi_filter_dropdown"));
   }
+  filterDropdownId () {
```
Comment (ldimaggi) [inline src/tests/work-item/work-item-list/page-objects/work-item-list.page.js]: This code is correct - but - why do we need 2 functions that access the same UI element?

Comment (naina-verma) [inline src/tests/work-item/work-item-list/page-objects/work-item-list.page.js]: This for just another way to test by different approach , Well when you manually go to Link item everything is clickable , So i have added multiple methods to verify that scenario!

Comment (ldimaggi) [inline src/tests/work-item/work-item-list/page-objects/work-item-list.page.js]: OK - sounds good - approved!

**Comment by nimishamukherjee** [comment]
[test]

**Pull request merged**